packages/astro/src/types/public/config.ts

```diff
@@ -287,14 +287,28 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 	 * You can customize the [redirection status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#redirection_messages) using an object in the redirect config:
 	 *
 	 * ```js
-	 * {
+	 * export default defineConfig({
 	 *   redirects: {
 	 *     '/other': {
 	 *       status: 302,
 	 *       destination: '/place',
 	 *     },
 	 *   }
-	 * }
+	 * })
+	 * ```
+	 *
+	 * Since **v5.2.0**, the property accepts external URLs:
```

Comment:
Note: See my suggestions in the Docs PR. I think this could nicely be incorporated into the existing description instead of a standalone section!